### DISCUSSION TITLE
Cannot move pages/change paths for pages

### DISCUSSION THREAD

**Discussion by cholesterol** [requarks/wiki#3705] [Error / Bug Report]
**Describe the bug**

When attempting to change the path of a page through the admin interface clicking the pencil icon doesn't do anything.  Trying to edit the path from within the page properties for the page itself also doesn't work.   I do remember this working in a previous release candidate so I'm not exactly sure what's going on here.  I have tried this with fresh pages as well as the pages I imported from my previous Wiki (via the git integration and an import), it does not work in either case. 

**To reproduce**

Attempt to edit the path for a page either through the admin interface or the page properties page

**Expected Behavior**

Clicking pencil icon should bring up the interface to edit the path for a page or saving a new path within the page properties should properly save the changes. 

**Host Info**

- OS: Ubuntu 14.04
- Wiki.js version: 2.0.12
- Database Engine: Postgresql version 9.6

**Comment by cholesterol** [comment]
Just adding to this - the "move rename" link from the standard page DOES work for moving, which is a great workaround atm but I think this should work elsewhere as well?

**Comment by Sousaplex** [comment]
I also had this issue when trying to correct the path to a page I had accidentally added a trailing slash.

**Comment by mpfaff** [comment]
@Cholesterol Am I supposed to be looking on the rendered wiki page? I don't see a "move rename" link there.
